src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_CleanGhost.java

```diff
@@ -54,7 +54,7 @@ public void onCommand(
 
         MsgUtil.sendMessage(sender,
                 ChatColor.GREEN
-                        + "Starting checking the shop be ghost, all does not exist shop will be removed...");
+                        + "->Starting checking the shop be ghost, all does not exist shop will be removed...");
 
```

Comment:
This line is notice send to server owner (no translable string) and all not exists shop will be removed (container block missing)